### ISSUE TITLE
package no longer builds

### ISSUE THREAD

**Issue by brittAnderson** [ntoronto/drbayes#8]
I realize the developer has probably moved on to other things, but I was wondering if there was any way to take a look to see what needs fixing for the package to build again? I am very new to racket, but interested in looking at it for a teaching tool and exploring probabilistic models. I really like the measure based approach. With some pointers I could try and see if I could get it compiling again. The error messages I see look to me mostly like naming clashes with other packages and nothing fundamentally outdated. If you suggest what I should read up on I will fork it and give a shot at updating. Thanks.

**Comment by soegaard** [comment]
Just as a spot check, I opened `real-tests/bijection-tests.rkt` in DrRacket and ran it.

The error message said, that in "private/flonum/probability.rkt" the identifier `list-set`
was imported twice. It seems `list-set` has been added to `racket/list` "recently",
so it clashes with the definition of `list-set` in DrBayes.

After changing    

    (require ... racket/list ...)

    (require ... (except-in racket/list list-set) ...)

I ran the code again, and got a similar error message.

After replacing `racket/list` with `(except-in racket/list list-set)` in "prob-set.rkt", "store.rkt", "store-set.rkt", "union.rkt"
and "union-set.rkt" the tests in "bijection-tests.rkt" run without any errors.

I am sure there are more files, that need the same treatment, but I don't think there are any major
problems getting DrBayes up to speed again.

**Comment by brittAnderson** [comment]
Thanks @soegaard . I will take a look at this over the next few days and see if I can fix things.

**Comment by soegaard** [comment]
@brittAnderson  I discovered that this issue is the same as issue #6 .
There is a pull request that solves the problem:

https://github.com/ntoronto/drbayes/pull/7/commits/b8e9bbb59cef319c09b427a663572c35b8a9feb0

It turns out, that the `list-set` used in DrBayes is the same as the one included in `racket/list`, which
means that the problem can be fixed by removing the definition of `list-set` in DrBayes (see PR7).

/Jens Axel

**Comment by brittAnderson** [comment]
Thanks for the update.Following your lead I looked at the src to see that the two `list-set` commands were similar, but the one from racket/list is untyped I believe, while the one in drbayes is typed. So rather than delete the drbayes version I did a find/replace `racket/list` with your `(except-in racket/list list-set)` on all files in drbayes on my local version of this package and it installs fine. 

First question: should we keep the typed version for use in drbayes (what I have done locally and in my fork) or would it be more racket idiomatic to use the racket/list "general" one as described in the prior pull request?

Second question: should I just maintain my own fork indefinitely or is there some other racketeer mechanism to pass useful packages on to new maintainers who can do these simple updates? And maybe given the low traffic this is not considered such a "useful" package? 

I will close this issue after giving @soegaard  a chance to opine.
